### PR TITLE
Ticket 2607: Galil specific opi

### PR DESCRIPTION
**Pull request by DominicOram**
### Description of work

This is an OPI that contains all the parameters that are specific to a galil over other motors.

### Ticket

https://github.com/ISISComputingGroup/IBEX/issues/2607

### Acceptance criteria

Will also need:
* https://github.com/ISISComputingGroup/EPICS-ioc/pull/203
* https://github.com/ISISComputingGroup/EPICS-galil/pull/16

To test:
* Point to a real galil 
* Confirm that all fields are connected and contain sensible values.
* Confirm the steps per unit labels change with updating motor units
* Confirm the inverse motor/encoder steps can be set

---

#### Code Review

- [ ] Is the code of an acceptable quality?
- [ ] Do the changes function as described and is it robust?
- [ ] Have the changes been documented in the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev)?

### Final Steps
- [ ] Reviewer has moved the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev) entry for this ticket in the "Changes merged into master" section

### CODE REVIEW

File: base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml [hunk 1]
```diff
@@ -1156,5 +1156,19 @@
 				</macros>
 			</value>
 		</entry>
+		<entry>
+			<key>Galil Engineering View</key>
```
Comment (Tom-Willemsen) [inline base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml]: I think this should be categorised under "miscellaneous motion control"

**Comment by Tom-Willemsen** [comment]
- [ ] I appear to be missing a file in `/resources/galil/single_axis.opi`... did you forget to commit it?
- [ ] In `galil.opi` the paths are currently hardcoded to look at `TE:NDLT702:...`

**Comment by DominicOram** [comment]
Apologies, should be better now. May also be worth checking with @KathrynBaker that there is nothing missing from this view. Motion control seem happy.

**Comment by Tom-Willemsen** [comment]
Merged for now because it meets the acceptance criteria of the original ticket. If @KathrynBaker has any suggestions it can be reopened (or a new ticket created).

**Pull request merged**